File: app/src/main/java/com/iomt/android/DeviceType.java
package com.iomt.android;

public class DeviceType {
    private String device_type;
    private String prefix;

    public DeviceType(String dt, String pr) {
        device_type = dt;
        prefix = pr;
    }

    public String getDevice_type() {
        return device_type;
    }

    public String getPrefix() {
        return prefix;
    }
}

File: app/src/main/kotlin/com/iomt/android/configs/DeviceType.kt
package com.iomt.android.configs

import com.iomt.android.R
import kotlinx.serialization.SerialName

/**
 * @property iconId
 */
enum class DeviceType(val iconId: Int) {
    @SerialName("bracelet") BRACELET(R.drawable.band_icon),
    UNKNOWN(R.drawable.default_device),
    @SerialName("vest") VEST(R.drawable.vest_icon),
    ;
}
